### DISCUSSION TITLE
I made a website to visualize 3D F1 data.

### DISCUSSION THREAD

**Discussion by adityakotha03** [br-g/openf1#31] [Show and tell]
Hi everyone,

I've been working on a web-based project that visualizes historic F1 race data in 3D. I was inspired by the recent VR trend and thought it would be cool to see race data in a more immersive way. For now, I've created a basic implementation using Three.js.

You can check out the current version here: github[(https://github.com/adityakotha03/F1_3D_Visualization)

Here's the deal: It's super basic right now. You pick a driver, select a timeframe, and bam! You see their car cruising along the track in 3D, all powered by data from the OpenF1 API.

I thought it'd be awesome to see if anyone here wants to jump in and play around with this project together. It's all in good fun, and who knows what cool features we could add? Right now, I'm thinking about stuff like:

1. A live-updating leaderboard
2. Hooking up to live data - imagine watching a race and having this 3D visualization running alongside it. Neat, right?

I'm just doing this for the kicks, no grand visions of startup glory or anything. Just thought it'd be a cool way to combine some of my interests and learn a bit along the way.

If you wanna see where it's at or have ideas on where it could go, let's chat in the comments.

[Screencast from 18-03-24 11:58:52 AM IST.webm](https://github.com/br-g/openf1/assets/86183042/cd2f4e5e-6c14-4416-bfd7-fa342c99a3b2)
![image](https://github.com/br-g/openf1/assets/86183042/f8ea9895-fb4a-4d28-8ce6-b8d4a126894c)
![image](https://github.com/br-g/openf1/assets/86183042/a850f6b4-ff5e-48b6-a14a-0f0e3b25dc37)

**Comment by br-g** [comment]
Very cool project, thanks for sharing!

**Comment by ant-dot-comm** [comment]
this is awesome! i was just searching around for inspiration to do something similar for fun a learning project too. happy to work with ya if the offer still stands. heres my last project for fun with threejs... https://ant-dot-comm.github.io/ant-grails-club/

**Reply by adityakotha03** [reply]
Yes I am very much excited to work together on this thing, your previous project looks amazing 🤩. Do you have discord. My username - pheonix0234.

**Reply by adityakotha03** [reply]
Or if there is any other msg platform you currently use so we can further discuss and manage there.

**Comment by EthanKyleKim** [comment]
I wonder how you created the track!

**Reply by adityakotha03** [reply]
https://github.com/adityakotha03/F1nsight/discussions/178

Hey take a look here, I have mentioned how we make the tracks in blender. The one in the video is just a png with colour turned white. But currently we use 3d maps made in blender. 
Thank you.

**Reply by EthanKyleKim** [reply]
Thank you!

**Comment by saiiimiiir** [comment]
Hello, your project is awesome ,i'm working on something similar but on meta quest 3 , take a look here [https://quickshare.samsungcloud.com/bF53g6V64xmT](url)
![5354b0d98761680f27d46c5ef2df1a0a](https://github.com/user-attachments/assets/dd9fcba7-fe04-4402-9cb6-83b01793645e)
![5354b0d98761680f27d46c5ef2df1a0a(1)](https://github.com/user-attachments/assets/ec57e274-eb12-4d7e-845d-40586a4deef1)

**Reply by adityakotha03** [reply]
This is freaking cool!!!!! Are you thinking of making this public?

**Reply by saiiimiiir** [reply]
thank you so much for your comment , yes I want to make this public but at least for the next F1 season or maybe later

**Reply by ant-dot-comm** [reply]
Yeah this is super awesome! Great work!

**Reply by saiiimiiir** [reply]
Thanks man appreciate it 🙏

**Reply by ant-dot-comm** [reply]
did you get a chance to check out what [adityakotha03](https://github.com/adityakotha03) and I created from his original post [F1nsight.com](https://www.f1nsight.com/#/race/1259).

**Reply by br-g** [reply]
Awesome!!

**Reply by saiiimiiir** [reply]
@ant-dot-comm yes I looked at the great work you've done on f1insight, so far I'm focused on flowing and managing data but surely I'll contact you when I want to make it all look better
